### PR TITLE
encode URI claims

### PR DESCRIPTION
**Pull request by susanharper-okta**
## Description:
- **What's changed?** added info on the support of encoded claims in URI format
- **Is this PR related to a Monolith release?** 2020.02.0

### Resolves:

* [OKTA-274467](https://oktainc.atlassian.net/browse/OKTA-274467)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/saml-hook/index.md [hunk 1]
```diff
@@ -368,6 +372,53 @@ When modifying an existing assertions statement, the path could begin with `/sub
 }
 ```
 
+## URL format response example
```
Comment (bellatumaneng-okta) [inline packages/@okta/vuepress-site/docs/reference/saml-hook/index.md]: `URI`

File: packages/@okta/vuepress-site/docs/reference/saml-hook/index.md [hunk 2]
```diff
@@ -368,6 +372,53 @@ When modifying an existing assertions statement, the path could begin with `/sub
...
+
+This example displays a `replace` and an `add` op with the URI formatted claim encoded.
```
Comment (bellatumaneng-okta) [inline packages/@okta/vuepress-site/docs/reference/saml-hook/index.md]: Seems like there should be a plural in here since there are two replace ops?

Comment (venkatviswanathan-okta) [reply packages/@okta/vuepress-site/docs/reference/saml-hook/index.md]: display `replace` and an `add` operations

Comment (susanharper-okta) [reply packages/@okta/vuepress-site/docs/reference/saml-hook/index.md]: That is true - there are two replace op and one add op in the example! GAH. I will update.

File: packages/@okta/vuepress-site/docs/reference/saml-hook/index.md [hunk 3]
```diff
@@ -368,6 +372,53 @@ When modifying an existing assertions statement, the path could begin with `/sub
...
+          "path": "/claims/http:~1~1schemas.xmlsoap.org~1ws~12005~105~1identity~1claims~1foo/attributes",
+          "value": {
+            "attributes": {
+              "NameFormat": "urn:oasis:names:tc:SAML:2.0:attrname-format:basic"
+            }
+          }
+        },
+        {
+          "op": "add",
+          "path": "/claims/http:~1~1schemas.xmlsoap.org~1ws~12005~105~1identity~1claims~1foo",
```
Comment (bellatumaneng-okta) [inline packages/@okta/vuepress-site/docs/reference/saml-hook/index.md]: Can we change the claim name to `http:~1~1schemas.xmlsoap.org~1ws~12005~105~1identity~1claims~1bar`? Just realize that this is trying to `replace` and `add` the same claim name 🤦‍♀

**Pull request merged**